src/test/scala/edu/ie3/mobsim/io/geodata/PointOfInterestSpec.scala

```diff
@@ -160,54 +161,6 @@ class PointOfInterestSpec extends UnitSpec with PoiTestData {
       }
     }
 
-    "find the nearest charging stations" should {
```

Comment:
One concern i might have is that we don't seem to test that we correctly add home charging stations to the home pois as in the "Parsing a whole file" test within the results the only home poi that is considered does not have an associated charging station. Maybe we could add one more that actually gets assigned to the home poi?

Reply:
Adapted the test so that a charging station each is now assigned to the home and the work poi